### PR TITLE
added get-topics endpoint and test

### PR DESCRIPTION
**Pull request by oregan1**
Previous request didn't get correctly merged so the changes are included in this request. (previous request was no.2 and was only changes to the readme.md)

### CODE REVIEW

**Comment by Dutton1990** [comment]
instead of setting your process.env.NODE_ENV to be 'test' if it isn't set, you want to run the seed function in your test file in a beforeEach. 

This way your test database will be seeded before each test and you still have the option of using the development database for hosting, which you wouldn't currently have.

Could you make that change please, push it up and put in another /nchelp pr so one of us can sign it off before merging :) cheers!

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            .then(({body}) => {
+                topics = body.topics;
+                expect(topics).toBeInstanceOf(Array);
+                topics.forEach((topic) => {
+                    expect(topic).toEqual(
+                        expect.objectContaining({
+                            slug: expect.any(String),
+                            description: expect.any(String)
+                        })
+                    );
```
Comment (emilyb93) [inline __tests__/app.test.js:31]: Id like to see some testing for the length of the array. A forEach testing an empty array will always pass so we should make sure that the array actually has things in it as well! :)

File: package.json [hunk 2]
```diff
@@ -21,14 +21,17 @@
...
-    "husky": "^7.0.0"
+    "pg-format": "^1.0.4"
   },
   "dependencies": {
     "dotenv": "^16.0.0",
-    "pg": "^8.7.3"
+    "express": "^4.18.1",
+    "pg": "^8.7.3",
+    "supertest": "^6.2.4",
+    "t": "^0.5.1"
```
Comment (emilyb93) [inline package.json:34]: possible accidental install here?

**Review by emilyb93 — CHANGES_REQUESTED** [review]
Test the length of the array, uninstall the 't' package and resubmit

File: __tests__/app.test.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ afterAll(() => {
 });
 
 
-describe('app - working', () => {
+describe.only('app - working', () => {
```
Comment (emilyb93) [inline __tests__/app.test.js:16]: Remove this .only and push back up and if it all goes well then feel free to merge

**Review by emilyb93 — APPROVED** [review]

**Pull request merged**